Userland/Libraries/LibC/assert.h

```diff
@@ -14,11 +14,11 @@ __BEGIN_DECLS
 __attribute__((noreturn)) void __assertion_failed(const char* msg);
 #    define __stringify_helper(x) #    x
 #    define __stringify(x) __stringify_helper(x)
-#    define assert(expr)                                                            \
-        (__builtin_expect(!(expr), 0)                                               \
-                ? __assertion_failed(#expr "\n" __FILE__ ":" __stringify(__LINE__)) \
-                : void(0))
-
+#    define assert(expr)                                                           \
+        do {                                                                       \
+            if (__builtin_expect(!(expr), 0))                                      \
+                __assertion_failed(#expr "\n" __FILE__ ":" __stringify(__LINE__)); \
+        } while (0)
```

Comment:
I'm not sure about this change. It's kind of sad to lose the nice property of it being a single expression.

Can we do something like a normal C-style cast? i.e.
```c
(void)0
```

Reply:
yep, revert is not gonna happen. "single expression" is more than just a nice property, it's effectively a requirement for portability. Fixed on master, please remove the commit.

Reply:
> I'm not sure about this change. It's kind of sad to lose the nice property of it being a single expression.

This has been fixed in master, going to update the PR
> Even if not, please change the commit message to match our guidelines: the first line should not be more than 72 characters long and it must start with a subsystem name.

I noticed this issue, but I was not sure about how the new name should be chosen.